runtime/moonbeam/tests/integration_test.rs

```diff
@@ -83,6 +83,7 @@ fn verify_pallet_prefixes() {
 	is_pallet_prefix::<moonbeam_runtime::AuthorFilter>("AuthorFilter");
 	is_pallet_prefix::<moonbeam_runtime::CrowdloanRewards>("CrowdloanRewards");
 	is_pallet_prefix::<moonbeam_runtime::AuthorMapping>("AuthorMapping");
+	is_pallet_prefix::<moonbeam_runtime::AuthorMapping>("MaintenanceMode");
```

Comment:
```suggestion
	is_pallet_prefix::<moonbeam_runtime::MaintenanceMode>("MaintenanceMode");
```

Reply:
woah, good catch! Why didn't this fail? :thinking:

Reply:
Oh, it did. Just not in ci because the build failed before that point. :relieved: